README.md

```diff
@@ -70,3 +70,88 @@ public class PlaySpringDIConfiguration {
 
 }
 ```
+
+## Migration Instructions
+
+If you want to migrate your existing project you should follow theese steps
+
+### For app without JPA
+
+1. Remove ```guice``` from your ```libraryDependencies``` in ```build.sbt``` file
+2. Make sure ```PlaySpringDIConfiguration``` is placed in your root package or add additional ```@ComponentScan``` annotations to it
```

Comment:
This also needs to be configured in `application.conf` as described above. It doesn't matter if it's in the root package. It just needs to be referenced in the configuration, e.g. `play.spring.configs += "com.example.myproject.PlaySpringDIConfiguration"`

Reply:
The fact that location of `PlaySpringDIConfiguration` must be mentioned in `application.conf` is in `Setup Instructions` so I think people will follow. But it's location is also important for spring. If you place it in the root of your project, you don't have to name specific packages in `@ComponentScan` - you can leave it empty, because Spring will scan all packages down the tree.
I'll change that to:
2. Make sure `PlaySpringDIConfiguration` is placed in your root package or you have to name specific packages in `@ComponentScan` annotation.